frontend/src/components/PageFooter.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
+        <footer>Copyright © Code for Poznan 2019</footer>
+    </div>
+</template>
+
+<script>
+export default {
```

Comment:
I guess <script> section is not required if component doesn't contain JS logic. Or is it (for more elegant code styling)? To be considered.